javascript/javascript3/week3/Promises/script.js

```diff
@@ -0,0 +1,51 @@
+//////////////////////////////////////////////////////Getting into promises
+
+
+
+// Promise.all(requests)
+//     .then(responses => {
+//         // all responses are resolved successfully
+//         responses.forEach(response => response.jcon())
+//     })
+const ul = document.querySelector('ul');
```

Comment:
It is nice to use a class name for the ul and use it here. it is not a recommended way to query because if later this page change by another developer, then it is not working correctly.